src/buffer/stm_tests.ml

```diff
@@ -25,7 +25,7 @@ struct
   type state = char list (* in reverse *)
   type sut = Buffer.t
 
-  let shrink_cmd c = match c with
+  let _shrink_cmd c = match c with
```

Comment:
In the other commits of that PR you removed the unused code.
You want to keep this one in case it turns out useful later?

Reply:
Yep, exactly :+1: